### PR TITLE
Add a tutorial to the readme

### PR DESCRIPTION
**Pull request by everlaat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,13 @@ An Actor Model implementation on top of the Elm Application Framework
...
 
-The documentation is hosted on the [Elm package website](https://package.elm-lang.org/packages/tricycle/elm-actor-framework/latest).
+Start with the (tutorial)(#Tutorial) in this Readme file.
+
+The API documentation is hosted on the [Elm package website](https://package.elm-lang.org/packages/tricycle/elm-actor-framework/latest).
 
-## Example Application
+## Example Applications
 
 There are three examples in the example folder.
```
Comment (alisa17) [inline README.md:16]: In order to **run**

File: README.md [hunk 2]
```diff
@@ -30,3 +31,405 @@ It's easy to run them locally;
...
+
+This let's us continue with the `factory` implementation; we were searching for
+a function that provides us with the required signature and by creating our
+`counter` `Actor` we have done just that.
+
+```elm
+
+type alias Msg = FrameworkMessage () () AppActors AppModel AppMsg
+
+factory: AppActors -> a -> (AppModel, Msg)
```
Comment (alisa17) [inline README.md]: I think this signature doesn't work? I think you need at least `factory : AppActors -> ( Pid, () ) -> ( AppModel, Msg )`

**Review by alisa17 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (everlaat) [reply README.md:16]: :+1:

File: README.md (hunk 2 above)
Comment (everlaat) [reply README.md]: :+1:

**Pull request merged**